### ISSUE TITLE
Add total pp rankings

### ISSUE THREAD

**Issue by respektive** [ppy/osu-web#8335]
Just to clarify by total pp i mean the sum of pp of all ranked plays a user has.
I think it would a cool addition in a sense that it's similar to the score rankings but also different. And because it's pp related, it might also just appeal a lot more to the general community than ranked score does. As lazer is getting closer to being complete and as far as i know, its not clear if the score rankings are getting removed, or if it's just gonna use the classic scoring, this might also be a good replacement for it, if it has to come to that. (i would prefer using classic scoring)

**Comment by peppy** [comment]
This is to be considered a new type "farm" leaderboard right? Because there's so many beatmaps that the people higher in this proposed leaderboard will be those that play the most, not those that are necessarily getting more pp-per-play.

**Comment by respektive** [comment]
yes that's correct. it would be mostly a board of who plays the most. however players that use more mods or get better scores still have a good chance of being on top. it's a mixture of both, which i think makes it pretty interesting.

i tried this out using the database dumps and xasuma was still on top while having over 10k less maps played than the player at rank 3.
